Changelog.md

```diff
@@ -0,0 +1,7 @@
+## [Unreleased] (07/01/23)
```

Comment:
This date looks like a pain to maintain every time we update.

Would it be easier to maintain the file without dates, or only dates for previous releases?

Reply:
agreed - done